source/database/restarting-elastic-search-service.html.md.erb

```diff
@@ -0,0 +1,45 @@
+---
+title: How to restart the Elastic Search service
+last_reviewed_on: 2024-11-04
+review_in: 12 months
+weight: 20
+---
+
+# <%= current_page.data.title %>
+
+There may be cases where the Elastic Search systemd service needs to be restarted on the VM. This guide provides instructions on how to restart this service.
+
+1) Request JIT access for non-prod or prod Bastion server depending on environment that is failing (https://myaccess.microsoft.com/)
+
+2) Connect to F5 VPN and log in to the appropriate Bastion server e.g. `bastion-nonprod.platform.hmcts.net`
+
+```
+az ssh config --ip \*.platform.hmcts.net --file ~/.ssh/config
+ssh bastion-nonprod.platform.hmcts.net
+```
+
+3) SSH into the Elastic Search node (there are generally 4 per environment)
+
+``` 
+ssh -i <logstash ssh key> elkadmin@<ip address>
+```	
+
```

Comment:
Not seeing this before I'm not sure which IP i'd choose in the env if there is 4, do you need to do this on each?

Reply:
Good spot, have added a line about checking each and the names of the VMs